railties/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+*   Sped up Docker build times by removing recrusive chown
```

Comment:
```suggestion
*   Sped up Docker build times by removing recursive chown
```

Reply:
Thanks! Good catch.

Reply:
This doesn't need a changelog

Reply:
@zzak Ah I see, dropped changelog entry!